e2e/runner/cypress-runner-run-tests.js

```diff
@@ -23,6 +46,13 @@ const configs = {
     const finalConfig = Object.assign({}, defaultConfig, userArgs);
     return finalConfig;
   },
+  "metabase-nodejs-react-sdk-embedding-sample-e2e": ({ suite }) =>
+    getSampleAppE2eConfig(suite),
+  "metabase-nextjs-sdk-embedding-sample-app-router-e2e": ({ suite }) =>
+    getSampleAppE2eConfig(suite),
+  "metabase-nextjs-sdk-embedding-sample-pages-router-e2e": ({ suite }) =>
+    getSampleAppE2eConfig(suite),
+  "shoppy-e2e": ({ suite }) => getSampleAppE2eConfig(suite),
```

Comment:
I don't see a reason for such indirection. These 4 suites have the same exact body.

I'd rather we hardcode the suite instead e.g.
```suggestion
  "metabase-nodejs-react-sdk-embedding-sample-e2e": () =>
    getSampleAppE2eConfig("metabase-nodejs-react-sdk-embedding-sample-e2e"),
  "metabase-nextjs-sdk-embedding-sample-app-router-e2e": () =>
    getSampleAppE2eConfig("metabase-nextjs-sdk-embedding-sample-app-router-e2e"),
  "metabase-nextjs-sdk-embedding-sample-pages-router-e2e": () =>
    getSampleAppE2eConfig("metabase-nextjs-sdk-embedding-sample-pages-router-e2e"),
  "shoppy-e2e": () => getSampleAppE2eConfig("shoppy-e2e"),
```

It makes more sense at this abstraction level. Otherwise, having 4 suites having the same config getter looks a bit confusing at glance.

Reply:
We can do:
const configs = {
   {...}
   ...getSampleAppE2eConfig("metabase-nodejs-react-sdk-embedding-sample-e2e"),
   ...getSampleAppE2eConfig("metabase-nextjs-sdk-embedding-sample-app-router-e2e"),
   ...getSampleAppE2eConfig("shoppy-e2e"),
   {...}
}

Reply:
that's better